src/test/policyestimator_tests.cpp

```diff
@@ -77,6 +84,13 @@ BOOST_AUTO_TEST_CASE(BlockPolicyEstimates)
         block.clear();
         // Check after just a few txs that combining buckets works as expected
         if (blocknum == 3) {
+            // Wait for fee estimator to catch up
```

Comment:
commit: Process fee estimates in the background via MempoolInterface

This code block crops up a lot. Move to a new function `std::future<void> ValidationInterfaceQueueClear()` (or some much better name that returns a future) in `validationinterface.h`.

Reply:
This was written before that existed. Fixed.